hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -277,7 +277,7 @@ protected void writeTableMetadata(HoodieTable table, String instantTime, String
    * Initialized the metadata table on start up, should only be called once on driver.
    */
   public void initMetadataTable() {
-    ((HoodieFlinkTableServiceClient<T>) tableServiceClient).initMetadataTable();
+    ((HoodieFlinkHoodieTableServiceClient<T>) tableServiceClient).initMetadataTable();
```

Comment:
redundant `Hoodie` in the name

Reply:
Fixed.

Reply:
it has the same problem with Java and Spark table service client classes. It should be HoodieJavaTableServiceClient and HoodieSparkRDDTableServiceClient

Reply:
the diagram needs update too